source/libraries/HelperFunctions.ts

```diff
@@ -74,14 +75,15 @@ export async function initializeTestRpcClientOptions(ethHttpProviderPort: number
     return { gasLimit: GAS_BLOCK_AMOUNT, accounts: accountOptions };
 }
 
-async function sleep(milliseconds: number): Promise<object> {
-    return new Promise((resolve, reject) => setTimeout(resolve, milliseconds));
+export async function sleep(milliseconds: number): Promise<void> {
+    return new Promise<void>((resolve, reject) => setTimeout(resolve, milliseconds));
 }
 
-export async function waitForTransactionToBeSealed(ethjsQuery: EthjsQuery, transactionHash: string): Promise<void> {
-    let transaction = await ethjsQuery.getTransactionByHash(transactionHash);
-    while (transaction.blockNumber == null) {
+export async function waitForTransactionReceipt(ethjsQuery: EthjsQuery, transactionHash: string): Promise<TransactionReceipt> {
+    let transactionReceipt = await ethjsQuery.getTransactionReceipt(transactionHash);
+    while (!transactionReceipt) {
         await sleep(ETHEREUM_POLLING_INTERVAL_MILLISECONDS);
-        transaction = await ethjsQuery.getTransactionByHash(transactionHash);
+        transactionReceipt = await ethjsQuery.getTransactionReceipt(transactionHash);
```

Comment:
Is what we really want to wait the transaction receipt, or for it to be mined? Since we've seen parity give receipts for un-mined TX. The contract_address will be correct, but an eth_call() won't have the data available to it yet

Reply:
Ah, good catch.  I meant to wait for the transaction receipt to have a block.  Will fix in next PR.